app/controllers/DashboardController.scala

```diff
@@ -54,7 +54,7 @@ trait DashboardController extends FrontendController with GatekeeperAuthWrapper
         val pendingVerification = grouped.getOrElse(PENDING_REQUESTER_VERIFICATION, Seq()) ++ grouped.getOrElse(PRODUCTION, Seq())
 
         Map("pendingApproval" -> pendingApproval.sortWith(ApplicationWithUpliftRequest.compareBySubmittedOn),
-            "pendingVerification" -> pendingVerification.sortWith(ApplicationWithUpliftRequest.compareBySubmittedOn))
+            "pendingVerification" -> pendingVerification.sortBy(_.name))
```

Comment:
Why are we using a map here, instead of a case-class or tuple.  We know at this point that the "pendingApproval" and "pendingVerification" keys are guaranteed to be here - which means that we can get a type-safe container that can be used with minimal fuss by saying: 

```scala
case class CategorisedApplications(
  pendingApproval: Seq[ApplicationWithUpliftRequest], pendingVerification: 
  Seq[ApplicationWithUpliftRequest])
```

And then using that here (and in the template) instead.